---
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: mbio.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 11pt
geometry: margin=1.0in
---

```{r knitr_settings, tidy=TRUE, eval=TRUE, echo=FALSE, cache=FALSE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}
	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```


\vspace{35mm}

# Proton pump inhibitor administration does not promote *Clostridium difficile* colonization in a murine model

\vspace{35mm}


Sarah Tomkovich${^1}$, Nicholas A. Lesniak${^1}$, Yuan Li${^1}$, Lucas Bishop${^1}$, Madison J. Fitzgerald${^1}$, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

\newpage
\linenumbers


## Abstract
Proton pump inhibitor (PPI) use has been associated with microbiota alterations and susceptibility to *Clostridium difficile* infections (CDIs) in humans. We assessed how PPI treatment alters the fecal microbiota and whether PPIs promote CDIs using a CDI mouse model. Mice were treated with the PPI omeprazole for 7 days prior to *C. difficile* challenge and were compared to mice that were treated with the antibiotic clindamycin or both omeprazole and clindamycin. We found that omeprazole was not sufficient to promote *C. difficile* colonization. When omeprazole treatment was combined with the antibiotic, there was a mixed level of *C. difficile* colonization, where one cage of mice remained resistant and the other cage was colonized. 16S rRNA sequencing analysis revealed that omeprazole had minimal impact on the murine microbiota throughout the 16 days of PPI exposure. *C. difficile* colonization of clindamycin-treated mice was associated with changes in *Alistipes*, *Barnesiella*, Porphyromonadaceae, and Ruminococcaceae. These results suggest PPI treatment alone is not sufficient to disrupt microbiota resistance to C. difficle infection in mice that are normally resistant in the absence of antibiotic treatment. 

## Importance
Antibiotics are a major risk factor for *Clostridium difficile* infections (CDIs), but it is less clear what other factors contribute to CDI incidence and recurrence. Interestingly, other medications besides antibiotics have been shown to alter the microbiota. Proton pump inhibitors (PPIs) are associated with alterations in the human intestinal microbiota through observational and interventional stduies and PPI use has also been associated with CDIs. We evaluated the ability of the PPI omeprazole to alter the murine intestinal microbiota and disrupt colonization resistance to *C. difficile*. We found PPI treatment had minimal impact on the murine intestinal microbiota and was not enough to promote *C. difficile* colonization after challenge. Further studies are needed to determine whether other factors such as the composition of the starting bacterial community, comorbidities, and use of additional medications contribute to the association between PPIs and CDIs.

\newpage

## Introduction
Proton pump inhibitors (PPIs) are one of the most widely used drugs and are available over the counter [@Singh2018]. PPIs are approved to treat a variety of conditions including gastroesophageal reflux disease, ulcers, or as prophylaxis for nonsteroidal anti-inflammatory drug (NSAID)-associated gastrointestinal bleeding [@nehra2018proton]. Unfortunately, there have been a number of adverse events associated with their use including small intestinal bacterial overgrowth, decreased calcium absorption, and increased risk of enteric infections [@nehra2018proton]. Thereâ€™s also been a number of large cohort studies as well as interventional clinical trials that have demonstrated specific alterations in the intestinal microbiome were associated with PPI use [@Imhann2017; @Naito2018]. PPI-associated microbiota changes have been attributed to the ability of PPIs to increase stomach acid pH which may promote the survival of oral and pathogenic bacteria [@Imhann2017; @Naito2018]. Human fecal microbiota changes with PPI use include increases in Enterococcaceae, Lactobacillaceae, Micrococcaceae, Staphylococcaceae and Streptococcaceae with decreases in Ruminococcaceae [@Imhann2017]. An *in vitro screen*, demonstrated that 3 PPIs inhibit the growth of 7-8 strains of Lachnospiraceae and Ruminococcaceae, which represented ~16% of the 40 intestinal bacteria strains that were screened [@maier2018extensive].  
Antibiotics have a large impact on the intestinal microbiome and are a primary risk factor for developing *Clostridium difficile* infections (CDIs) [@Schubert2015]. It is less clear whether other human medications that impact the microbiota also influence *C. difficile* colonization resistance. Multiple epidemiological studies have suggested an association between PPI use and incidence or recurrence of CDIs [@tariq2017association; @nehra2018proton; @Naito2018; @Elias2019]. However, most of the studies suggesting a link between PPIs and *C. difficile* are retrospective and did not evaluate the microbiome [@tariq2017association; @nehra2018proton; @Elias2019]. Thus, it is unclear whether the gastrointestinal microbiome changes associated with PPI use play a role in the association between PPIs and CDI incidence or recurrence. Additionally, epidemiological studies have a limited capacity to address potential confounders and comorbidities in patients that were on PPIs and developed CDIs or recurrent CDIs [@Elias2019; @tariq2017association].    
Although there have been a few *C. difficile* mouse model studies that have demonstrated PPIs have some effect on CDIs with or without additional antibiotic treatment, none of these studies looked at the interplay with the intestinal microbiome [@hung2015proton; @Kaur2007; @kaur2011effect]. Here, we evaluated the impact of a PPI on the murine microbiome and susceptibility to *C. difficile* colonization in relation to clindamycin, an antibiotic that perturbs the microbiome enough to allow *C. difficile* to colonize but is mild enough that *C. difficile* is cleared within 10 days [@Jenior2018].

## Results

**Murine fecal microbiomes were minimally affected by PPI treatment **
To test if PPI treatment alters the microbiome and promotes susceptibility to CDIs, we gavaged mice daily with a high dose of omeprazole for 7 days before *C. difficile* challenge (Figure 1A) and compared the PPI-treated mice to a group that received the antibiotic, clindamycin, and a group that received both PPI and clindamycin treatment. A principle coordinates analysis (PCoA) of the Bray-Curtis distances over the initial 7 days of treatment revealed the bacterial communities of PPI-treated mice remained relatively unchanged (Figure 1B). Enterococcaceae, Lactobacillaceae, Micrococcaceae, Staphylococcaceae, Streptococcaceae, and Ruminococcaceae are all familes that have previously been impacted by PPIs in human studies, but we see no significant fluctuations in the stool of our PPI-treated mice throughout the course of the 16-day experiment (Figure 1C). In contrast, the microbiomes from the 2 groups of mice that received clindamycin started to shift away from the untreated and PPI-treated mice the day after treatment (Figure 1D).

**PPI-treatment did not promote susceptibility to *C. difficile infection* in mice**
After 7 days of PPI treatment or 1 day after clindamycin treatment, mice were challenged with *C. difficile* 630 spores. While all 4 of the clindamycin-treated mice were colonized with *C. difficile*, all of the PPI-treated mice were resistant to *C. difficile* colonization (Figure 2A). Interestingly, only 1 cage of the PPI and clindamycin-treated mice were colonized, while the other cage was not (Figure 2A). PCoAs of the bacterial communities from the 3 groups of mice after *C. difficile* challenge, showed that the greatest shifts in bacterial communities occurred in the clindamycin-treated mice (Figure 2B). Within 5 days, all of the mice had cleared *C. difficile*, suggesting there was no difference in rate of clearance for the mice that were initially colonized with *C. difficile* despite continuing to receive PPI treatment throughout the course of the experiment. Our results suggest that PPI treatment alone had minor effects on microbial community resistance to *C. difficile* colonization in mice. Instead most of the differences between our 3 treatment groups appear to be driven by clindamycin administration and included decreased *Alistipes*, *Barnesiella*, Porhyromonadaceae, Ruminococcaceae, taxa previously found to be altered in clindamycin-treated mice that were challenged with *C. difficile* [@Schubert2015].

## Discussion and Conclusions

Our results demonstrate the PPI omeprazole had minimal impact on the stool microbiomes of mice and did not promote susceptibility to *C. difficile* colonization. In contrast, clindamycin-treated mice were initially colonized, but cleared the infection within 10 days. Combining PPI and clindamycin treatment had a mixed effect, one cage was colonized with *C. difficile* while the other cage remained resistant.  
Our findings that PPI treatment had minimal impact on the fecal microbiome are comparable to another PPI mouse study that indicated PPIs have more of an effect on the small intestinal microbiota compared to the fecal microbiota [@Yasutomi2018]. We did not find significant changes for the taxa observed to be significantly impacted by PPI use in human studies, but a closer examination suggests that the significance of these associations in humans is driven by some individuals more than others and the effect size can be relatively small with overall differences on a PCoA plot difficult to distinguish [@Freedberg2015, @Imhann2015]. Also, most of the studies to date compare different individuals (PPI users versus non-users) or the same individuals before and after treatment [@Naito2018; @Imhann2017], unfortunately these limited microbiota snapshots may be skewed by day-to-day microbiota variations [@LlornsRico2019] that would be revealed with more frequent longitudinal sampling.   
Previous work that administered the PPI lansoprazole daily for 2 weeks to mice and then challenged with *C. difficile* demonstrated that PPI treatment alone resulted in detectable *C. difficile* in the stool 1 week after challenge, but also showed there was detectable *C. difficile* in mice not treated with antibiotics [@Kaur2007; @kaur2011effect]. The presence of *C. difficile* in mice that were not treated with either antibiotics or PPIs could be attributed to the higher dose of vegetative cells (10^8^ CFU) and the different *C. difficile* strain used to challenge the mice and may partly explain their observation of *C. difficile* in PPI-treated mice [@Kaur2007; @kaur2011effect]. We have previously shown mice from are colony that are not given antibiotics are resistant to *C. difficile* 630 when challenged with 10^3^ spores [@Jenior2017]. Another group also used a higher dose of 10^6^ spores or 10^7^ vegetative cells and a different *C. difficile* strain to challenge antibiotic-treated mice or mice treated with both the PPI esomeprazole for 2 days and antibiotics and demonstrated the antibiotic/PPI-treated mice developed more severe CDIs [@hung2015proton].  
Our study extended previous work examining PPIs and *C. difficile* in mice to examine the contribution of the intestinal microbiota. We found the PPI omeprazole had minimal impact on the murine intestinal microbiota and in contrast to previous work, PPIs did not promote *C. difficile* colonization. Our 16S rRNA sequencing suggested that *Streptococcus* and *Enterococcus* are rare taxa in our C57BL/6 mouse colony. If these taxa are important contributors to the associations between PPIs and CDIs in humans, this may be part of the reason we did not see an effect in our CDI mouse model. There could also be differences at the species level for taxa that are altered with PPIs in humans compared to mice, which may also contribute to the association. Other murine starting communities or the addition of human PPI treatment-associated strains may be needed to determine whether PPIs can alter taxa and subsequently promote CDIs.  
Factors such as age, body mass index, comorbidities, and use of other medications in human studies may also be contributing to the association between PPIs and CDIs. NSAID treatment can increase CDI severity in mice [@Maseda2019]. Given that PPIs are sometimes used in combination with NSAIDs [@nehra2018proton], it may be worth examining both medications together using a mouse model. This study only looked at the impact of PPIs with or without antibiotics, but future work may want to incorporate age, other comorbidities and bacterial strains that are less common in mice to evaluate whether PPIs in conjunction with other factors can increase the risk of CDIs.   

## Materials and Methods

**Animals** All mouse experiments were performed with 7- to 12-week-old C57BL/6 male and female mice. 2-3 mice were housed per cage and male and female mice were housed separately. All animal experiments were approved by the University of Michigan Animal Care and Use Committee (IACUC) under protocol number PRO00006983.

**Drug treatments**
Omeprazole (Sigma Aldrich) was prepared in a vehicle solution of 40% polyethylene glycol 400 (Sigma-Aldrich) in phosphate buffered saline. Omeprazole was prepared from 20 mg/mL frozen aliquots and diluted to an 8mg/mL prior to gavage. All mice received ~40 mg/kg omeprazole or vehicle solution once per day through the duration of the experiment with treatment starting 7 days before *C. difficile* challenge (Figure 1A). One day prior to *C. difficile* challenge, mice received an intraperitoneal injection of 10 mg/kg clindamycin or sterile saline vehicle. All drugs were filter sterilized through a 0.22 micron syringe filter before administration to animals.

**C. difficile infection model**
Mice were challenged with *C. difficile* 630 seven days after the start of omeprazole treatment and one day after clindamycin treatment. Mice were challenged with 10^3^ spores in ultrapure distilled water. Samples were collected for 16S rRNA sequencing or *C. difficile* CFU quantification throughout the duration of the experiments at the indicated timepoints (Figure 1A). Samples for 16S rRNA sequencing were flash frozen in liquid nitrogen and stored at -80&deg;C until DNA extraction, while samples for CFU quantification were transferred into an anaerobic chamber and serially diluted in PBS. Diluted samples were plated on TCCFA (taurocholate, cycloserine, cefoxitin, fructose agar) plates and incubated at 37&deg;C for 24 hours under anaerobic conditions to quantify *C. difficile* CFU.

**16S rRNA gene sequencing** DNA for 16S rRNA gene sequencing was extracted from 10-50 mg fecal pellet from each mouse using the DNeasy Powersoil HTP 96 Kit (Qiagen) and an EpMotion 5075 automated pipetting system (Eppendorf). The V4 hypervariable region of the 16S rRNA gene was amplified with Accuprime Pfx DNA polymerase (Thermo Fisher Scientific) using previously described custom barcoded primers [@Kozich2013]. The 16S rRNA amplicon library was sequenced with the MiSeq (Illumina). Amplicons were cleaned up and normalized with the SequalPrep Normalization Plate Kit (ThermoFisher Scientific) and pooled amplicons were quantified with the KAPA library quantification kit (KAPA Biosystems).

**16S rRNA gene sequence analysis**Mothur (v1.40.5) was used for all sequence processing steps [@Schloss2009] using the previously published protocol [@Kozich2013].  R (v.3.5.1) was used to generate figures and perform statistical analysis.

**Statistical Analysis**

**Code availability** The code for all sequence processing and analysis step as well as an Rmarkdown version of this manuscript is available at https://github.com/SchlossLab/Tomkovich_PPI_XXXX_2019.

**Data availability** The 16S rRNA sequencing data have been deposited in the NCBI Sequence Read Archive (Accession no. _________)

\newpage

## Figures

**Figure 1. PPI treatment had minimal impact on the murine fecal microbiota** A. Mouse experiment timeline and logistics. B. Principal Coordinates Analaysis (PCoA) of Bray-curtis distances from stool samples for all treatment groups during the intial 7 days of treatment with the PPI omeprazole or vehicle cluster together and do not chage much over time. C. Relative abundances of families previously associated with PPI use in humans do not change much over the 16-day course of treatment with PPIs (7 days before *C. difficile* challenge through 9 days post *C. difficile* challenge). D. PCoA of stool samples collected before and 1 day after antibiotic treatment, demonstrating the fecal microbiota of mice treated with clindamycin starts to separate from the rest of the samples.

**Figure 2. PPI treatment alone does not promote CDIs in mice** A. C. difficile CFUs/g stool measured each day post C. difficile challenge for clindamycin, clindamycin/PPI, and PPI-treated mice. B. PCoA of stool samples collected after antibiotic treatments. C. Genera significantly associated with treatment groups. D. Families significantly associated with treatment groups.


\newpage

## References
