---
bibliography: references.bib
csl: mbio.csl
fontsize: 11pt
geometry: margin=1.0in
output:
  word_document: default
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
---

```{r knitr_settings, tidy=TRUE, eval=TRUE, echo=FALSE, cache=FALSE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}
	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```


\vspace{35mm}

# The proton pump inhibitor omeprazole does not promote *Clostridium difficile* colonization in a murine model

\vspace{35mm}


Sarah Tomkovich${^1}$, Nicholas A. Lesniak${^1}$, Yuan Li${^1}$, Lucas Bishop${^1}$, Madison J. Fitzgerald${^1}$, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

\newpage
\linenumbers


## Abstract
Proton pump inhibitor (PPI) use has been associated with microbiota alterations and susceptibility to *Clostridium difficile* infections (CDIs) in humans. We assessed how PPI treatment alters the fecal microbiota and whether treatment promotes CDIs in a mouse model. Mice receiving a PPI treatment were gavaged with 40 mg/kg of omeprazole during a 7-day pretreatment phase, the day of *C. difficile* challenge, and the following 9 days. We found that mice treated with omeprazole were not colonized by *C. difficile*. When omeprazole treatment was combined with a single clindamycin treatment, one cage of mice remained resistant to *C. difficile* colonization, while the other cage was colonized. Treating mice with only clindamycin followed by challenge resulted in  *C. difficile* colonization. 16S rRNA gene sequencing analysis revealed that omeprazole had minimal impact on the structure of the murine microbiota throughout the 16 days of omeprazole exposure. These results suggest omeprazole treatment alone is not sufficient to disrupt microbiota resistance to *C. difficle* infection in mice that are normally resistant in the absence of antibiotic treatment. 

## Importance
Antibiotics are the primary risk factor for *Clostridium difficile* infections (CDIs), but other factors may also increase a person's risk. In epidemiological studies, PPI use has been associated with CDI incidence and recurrence. Proton pump inhibitors (PPIs) have also been associated with alterations in the human intestinal microbiota in observational and interventional studies. We evaluated the effects of the PPI omeprazole on the structure of the murine intestinal microbiota and its ability to disrupt colonization resistance to *C. difficile*. We found omeprazole treatment had minimal impact on the murine fecal microbiota and did not promote *C. difficile* colonization. Further studies are needed to determine whether other factors such as the composition of the starting bacterial community, comorbidities, and use of additional medications contribute to the association between PPIs and CDIs seen in humans or whether aspects of murine physiology may limit its utility to test these types of hypotheses.

\newpage

Antibiotics have a large impact on the intestinal microbiome and are a primary risk factor for developing *Clostridium difficile* infections (CDIs) [@Schubert2015]. It is less clear whether other human medications that impact the microbiota also influence *C. difficile* colonization resistance. Multiple epidemiological studies have suggested an association between proton pump inhibitor (PPI) use and incidence or recurrence of CDIs [@tariq2017association; @nehra2018proton; @Naito2018; @Elias2019]. There have also been a number of large cohort studies and interventional clinical trials that demonstrated specific alterations in the intestinal microbiome were associated with PPI use [@Imhann2017; @Naito2018]. PPI-associated microbiota changes have been attributed to the ability of PPIs to increase stomach acid pH which may promote the survival of oral and pathogenic bacteria [@Imhann2017; @Naito2018]. In human fecal samples, PPI use results in increases in Enterococcaceae, Lactobacillaceae, Micrococcaceae, Staphylococcaceae and Streptococcaceae and decreases in Ruminococcaceae [@Imhann2015; @Imhann2017; @Freedberg2015; @maier2018extensive]. Several of these taxa have also been associated with *C. difficile* colonization in humans [@Schuberte01021-14].      
Unfortunately, most of the studies suggesting a link between PPIs and *C. difficile* were retrospective and did not evaluate changes in the microbiome [@tariq2017association; @nehra2018proton; @Elias2019]. Thus, it is unclear whether the gastrointestinal microbiome changes associated with PPI use explain the association between PPIs and CDIs. Additionally, epidemiological studies have a limited capacity to address potential confounders and comorbidities in patients that were on PPIs and developed CDIs or recurrent CDIs [@Elias2019; @tariq2017association].  Here, we evaluated the impact of daily PPI treatment with omeprazole on the murine microbiome and susceptibility to *C. difficile* colonization in relation to clindamycin, an antibiotic that perturbs the microbiome enough to allow *C. difficile* to colonize but is mild enough that *C. difficile* is cleared within 10 days [@Jenior2018].

**Murine fecal microbiomes were minimally affected by omeprazole treatment ** To test whether omeprazole treatment alters the microbiome and promotes susceptibility to CDIs, we gavaged mice with 40 mg/kg of omeprazole for 7 days before *C. difficile* challenge (Figure 1A).  A principle coordinates analysis (PCoA) of the Bray-Curtis distances over the initial 7 days of treatment revealed the bacterial communities of omeprazole-treated mice remained relatively unchanged (Figure 1B). We observed no significant fluctuations in the relative abundance of those taxa previously shown to respond to PPI treatment throughout the course of the 16-day experiment (Figure 1C-D, S1). We also observed no significant changes in relative abundances at the family and genus level over the course of the experiment for the omeprazole-treated mice (data not shown). These results demonstrated that the omeprazole treatment alone had a minimal impact on the murine fecal bacterial community after 7 days of pretreatment.

**Omeprazole treatment did not promote susceptibility to *C. difficile infection* in mice** Next, we examined whether omeprazole treatment altered susceptibility to *C. difficile* infection in mice. After omeprazole treatment or clindamycin treatment, mice were challenged with 10^3^ *C. difficile* 630 spores. While *C. difficile* colonized the clindamycin-treated mice, it did not colonize the omeprazole-treated mice (Figure 2A). Interestingly, only 1 cage of mice that received both omeprazole and clindamycin were colonized, while the other cage of mice were resistant (Figure 2A). The greatest shifts in bacterial communities occurred in the clindamycin-treated mice (Figure 2B, S2). Regardless of whether the mice became colonized, all of the mice had cleared *C. difficile* within 5 days (Figure 2A), suggesting that omeprazole did not affect the rate of clearance. Our results suggest that omeprazole treatment had no effect on bacterial community resistance to *C. difficile* colonization in mice. Instead most of the differences between the 3 treatment groups appeared to be driven by clindamycin administration (Figure 2C, S2). These findings demonstrated that high dose omeprazole treatment did not promote susceptibility to *C. difficile* colonization.

## Conclusions
We found the PPI omeprazole did not impact the gut microbiota and did not promote *C. difficile* infection in mice. Our findings that omeprazole treatment had minimal impact on the fecal microbiome were comparable to another PPI mouse study that indicated the PPI lansoprazole had more of an effect on the small intestinal microbiota compared to the fecal microbiota [@Yasutomi2018]. The same group demonstrated lansoprazole treatment increased the stomach pH in mice [@Yasutomi2018], which may improve survival of bacteria passing through the stomach. We did not find significant changes for the taxa observed to be significantly impacted by PPI use in human studies. However, 3 of the human-associated taxa were absent or at low abundance in our mice. Although our fecal microbiota findings are comparable to what has been shown in another mouse study [@Yasutomi2018], whether PPI-induced changes in specific bacterial abundances observed in humans play a role in CDIs remains to be determined.    
While several *C. difficile* mouse model studies have shown that PPIs have an effect on CDIs with or without additional antibiotic treatment [@Kaur2007; @kaur2011effect; @hung2015proton], there were insufficient controls to attribute the effect solely to PPI treatment. One group administered 0.5 mg/kg of the PPI lansoprazole daily for 2 weeks to mice and then challenged with *C. difficile* demonstrated that PPI treatment alone resulted in detectable *C. difficile* in the stool 1 week after challenge, however there was detectable *C. difficile* in mice not treated with antibiotics [@Kaur2007; @kaur2011effect]. The other mouse study demonstrated the antibiotic/PPI (esomeprazole)-treated mice developed more severe CDIs compared to antibiotic-treated mice, but did not have a group treated with just esomeprazole for comparison [@hung2015proton]. We tested the same high 40 mg/kg PPI dose and expanded pre-treatment to 7 days before challenge to test the impact of omeprazole treatment alone on our CDI mouse model. Additionally, we've previously demonstrated that mice from our colony were resistant to *C. difficile* 630 colonization without antibiotic treatment [@Jenior2017], ensuring there was not already partial susceptiblity to *C. difficile* before treatment. The additional controls in our study allowed us to assess the contibution of omeprazole alone to *C. difficile* susceptibility in mice.  
Our study also extended previous work examining PPIs and *C. difficile* in mice by incorporating the contribution of the intestinal microbiota. We found the PPI omeprazole had no significant impact on bacterial taxa within the murine intestinal microbiota over the 16-day experiment. In contrast to previous work, omeprazole did not alter *C. difficile* colonization resistance. 16S rRNA sequencing suggested that *Streptococcus* and *Enterococcus* are rare genera in our C57BL/6 mouse colony and variation in these 2 genera have been observed across other facilities and vendors [@Rausch2016; @Robertson2019]. These genera could be important contributors to the associations between PPIs and CDIs in humans, and could be a contributing factor to our observation that PPI treatment had no effect on *C. difficile* colonization in our CDI mouse model. Gastrointestinal physiological differences, particularly the higher stomach pH in mice (pH 3-4) compared to humans (ph 1) [@Hugenholtz2017] could also explain why omeprazole had a limited impact on the murine microbiome. The microbiota and physiological differences between humans and mice may limit the usefulness of employing mouse models to study the impact of PPIs on the microbiota and CDIs.   
Beyond microbiome differences, factors such as age, body mass index, comorbidities, and use of other medications in human studies may also be contributing to the association between PPIs and CDI incidence or recurrence. This study addressed the impact of PPIs with or without antibiotics on a murine model of CDI, and found PPIs did not promote *C. difficile* colonization. The epidemiological evidence linking PPIs to CDIs is primarily from observational studies, which makes determining causality and whether other risk factors play a role challenging [@Eze2017]. Future studies are needed to determine whether age, other comorbidities and bacterial strains that are less common in mice can increase the risk of CDIs or recurrent CDIs when combined with PPI treatment.   

## Acknowledgements

This research was supported by NIH grant U01AI12455. We would also like to thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the infrastructure and support for performing our mouse expeirments. The authors are also thankful to members of the Schloss lab for helpful discussions throughout the process of designing the experiment, analyzing the results, crafting the figures, and drafting of the manuscript.  

\newpage

## Materials and Methods

**Animals.** All mouse experiments were performed with 7- to 12-week-old C57BL/6 male and female mice. Each experimental group of mice was spit between 2 cages with 2-3 mice housed per cage and male and female mice housed separately. All animal experiments were approved by the University of Michigan Animal Care and Use Committee (IACUC) under protocol number PRO00006983.

**Drug treatments.**
Omeprazole (Sigma Aldrich) was prepared in a vehicle solution of 40% polyethylene glycol 400 (Sigma-Aldrich) in phosphate buffered saline. Omeprazole was prepared from 20 mg/mL frozen aliquots and diluted to an 8 mg/mL prior to gavage. All mice received 40 mg/kg omeprazole (a dose previously used in mouse experiments [@hung2015proton]) or vehicle solution once per day through the duration of the experiment with treatment starting 7 days before *C. difficile* challenge (Figure 1A). One day prior to *C. difficile* challenge, 2 groups of mice received an intraperitoneal injection of 10 mg/kg clindamycin or sterile saline vehicle [@Jenior2018]. All drugs were filter sterilized through a 0.22 micron syringe filter before administration to animals.

***C. difficile* infection model.**
Mice were challenged with *C. difficile* 630 seven days after the start of omeprazole treatment and one day after clindamycin treatment. Mice were challenged with 10^3^ spores in ultrapure distilled water as described previously [@Jenior2018]. Stool samples were collected for 16S rRNA sequencing or *C. difficile* CFU quantification throughout the duration of the experiments at the indicated timepoints (Figure 1A). Samples for 16S rRNA sequencing were flash frozen in liquid nitrogen and stored at -80&deg;C until DNA extraction, while samples for CFU quantification were transferred into an anaerobic chamber and serially diluted in PBS. Diluted samples were plated on TCCFA (taurocholate, cycloserine, cefoxitin, fructose agar) plates and incubated at 37&deg;C for 24 hours under anaerobic conditions to quantify *C. difficile* CFU.

**16S rRNA gene sequencing.** DNA for 16S rRNA gene sequencing was extracted from 10-50 mg fecal pellet from each mouse using the DNeasy Powersoil HTP 96 Kit (Qiagen) and an EpMotion 5075 automated pipetting system (Eppendorf). The 16S rRNA sequencing libary was prepared as described previously [@Kozich2013]. In brief, the ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used as a mock community [@Sze2019] and water was used as a negative control. The V4 hypervariable region of the 16S rRNA gene was amplified with Accuprime Pfx DNA polymerase (Thermo Fisher Scientific) using previously described custom barcoded primers [@Kozich2013]. The 16S rRNA amplicon library was sequenced with the MiSeq (Illumina). Amplicons were cleaned up and normalized with the SequalPrep Normalization Plate Kit (ThermoFisher Scientific) and pooled amplicons were quantified with the KAPA library quantification kit (KAPA Biosystems).

**16S rRNA gene sequence analysis.** mothur (v1.40.5) was used for all sequence processing steps [@Schloss2009] using a previously published protocol [@Kozich2013]. In brief, forward and reverse reads for each sample were combined and low-quality sequences and chimeras were removed. Duplicate sequences were merged, before taxonomy assignment using a modified version (v16) of the Ribosomal Database Project reference database (v11.5) with an 80% cutoff. Operational taxonomic units (OTUs) were assigned with the opticlust clustering algorithm using a 97% similarity threshold. To adjust for uneven sequencing across samples, all samples were rarefied to 3,000 sequences, 1,000 times. PCoAs were generated based on Bray-Curtis distance. R (v.3.5.1) was used to generate figures and perform statistical analysis. 

**Statistical Analysis.** To test for differences in relative abundances in families and genera across our 3 different treatment groups (Clindamycin, Clindamycin + PPI, and PPI) or within the PPI treatment group across 3 timepoints (Day -7, 0, and 9), we used a Kruskal-Wallis test with a Benjamini-Hochberg correction for multiple comparisons.

**Code availability.** The code for all sequence processing and analysis step as well as an Rmarkdown version of this manuscript is available at https://github.com/SchlossLab/Tomkovich_PPI_mSphere_2019.

**Data availability.** The 16S rRNA sequencing data have been deposited in the NCBI Sequence Read Archive (Accession no. PRJNA554866).

\newpage

## References

<div id="refs"></div>

\newpage

## Figures
\includegraphics{figure_1.pdf}
**Figure 1. Omeprazole treatment had minimal impact on the murine fecal microbiota.** A. Mouse experiment timeline and logistics. The PPI omeprazole was administered throughout the duration of the experiment. Clindamycin was administered 1 day before *C. difficile* challenge on Day 0. Stools for 16S rRNA sequencing analysis were collected on the days that are labeled (Day -7, -5, -3, -1, 0, 1, 2, 3, 4, 5, 7, 9). *C. difficile* CFU in the stool was quantified daily through 6 days post-infection by anaerobic culture. B. Principal Coordinates Analaysis (PCoA) of Bray-curtis distances from stool samples of mice in the omeprazole treatment group during the intial 7 days of the experiment. Each color represents stool samples from the same mouse and lines connect sequentially collected samples. C-D. Relative abundances of families previously associated with PPI use in humans at the start of the experiment (C) and after 7 days of omeprazole treatment (D). Each circle represents an individual mouse. There were no significant differences acoss treatment groups for any of the identified families in the seqeunce data at day -7 and day 0, analyzed by Kruskal-Wallis test with a Benjamini-Hochberg correction for multiple comparisons. For C-D, the grey vertical line indicates the limit of detection.

\newpage
\includegraphics{figure_2.pdf}
**Figure 2. Omeprazole treatment alone does not promote CDIs in mice.** A. *C. difficile* CFUs/g stool measured each day post *C. difficile* challenge for clindamycin, clindamycin/PPI, and PPI-treated mice. Lines represent the mean CFU/g for each treatment group while points represent CFU/g for individual mice within each group. The black dashed line indicates the limit of detection. B. PCoA of of Bray-curtis distances from stool samples collected after antibiotic treatment (last 9 days of the experiment). Transparency of the symbol corresponds to treatment day. Symbols represent the *C. difficile* colonization status of the mice measured 2 days post-infection. Circles represent resistant mice (*C. difficile* was undetectable in stool samples), while X-shapes represent mice that were colonized with *C. difficile*, although all mice cleared *C. difficile* within 5 days of infection. Omeprazole treated fecal samples primarily cluster together throughout the experiment. C. Genera that vary the most across treatment groups for stool samples collected from mice 2 days post-infection. Data were analyzed by Kruskal-Wallis test, and no P values were significant after Benjamini-Hochberg correction for multiple comparisons. The grey vertical line indicates the limit of detection.

\newpage
\includegraphics{figure_s1.pdf}
**Figure S1. Families within omeprazole treated mice fluctuate over time with no overall trend in either direction.** Relative abundance over time for Lactobacillaceae (A) and Ruminococcaceae (B), 2 of the PPI-associated familes from human PPI studies across all 3 treatment groups. Each point represents the relative abundance for an individual mouse stool sample, while the lines represent the mean relative abundances for each treatment group of mice. The grey horizontal lines indicate the limit of detection.

\newpage
\includegraphics{figure_s2.pdf}
**Figure S2. Microbiota diversity and richness decrease with antibiotic treatment but remain relatively constant with omeprazole treatment.** A. Boxplots of the Shannon Diversity Index values (A) and number of observed OTUs (B) for each group of mice over 3 timepoints (Day -7, 0, and 9). Each circle represents the value for a stool sample from an individual mouse. 
