---
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: mbio.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 11pt
geometry: margin=1.0in
---

```{r knitr_settings, tidy=TRUE, eval=TRUE, echo=FALSE, cache=FALSE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}
	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```


\vspace{35mm}

# Proton pump inhibitor administration does not promote *Clostridium difficile* colonization in a murine model

\vspace{35mm}


Sarah Tomkovich${^1}$, Nicholas A. Lesniak${^1}$, Yuan Li${^1}$, Lucas Bishop${^1}$, Madison J. Fitzgerald${^1}$, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

\newpage
\linenumbers


## Abstract
Proton pump inhibitor (PPI) use has been associated with microbiota alterations and susceptibility to *Clostridium difficile* infections (CDIs) in humans. We assessed how PPI treatment alters the fecal microbiota and whether PPIs promote CDIs using a CDI mouse model. Mice were treated with a high daily dose of the PPI omeprazole for 7 days prior to *C. difficile* challenge and were compared to mice that were treated with the antibiotic clindamycin or both omeprazole and clindamycin. High-dose PPI treatment was maintained throughout the experiment, which included pre-treatment, the day of *C. difficile* challenge, and the following 9 days. We found that omeprazole was not sufficient to promote *C. difficile* colonization. When omeprazole treatment was combined with the antibiotic,one cage of mice remained resistant to *C. difficile* colonization, while the other cage was colonized. 16S rRNA sequencing analysis revealed that omeprazole had minimal impact on the murine microbiota throughout the 16 days of PPI exposure. These results suggest PPI treatment alone is not sufficient to disrupt microbiota resistance to *C. difficle* infection in mice that are normally resistant in the absence of antibiotic treatment. 

## Importance
Antibiotics are a major risk factor for *Clostridium difficile* infections (CDIs), but other factors may also contribute to CDI incidence and recurrence. Interestingly, other medications besides antibiotics can impact the microbiota. Proton pump inhibitors (PPIs) are associated with alterations in the human intestinal microbiota through observational and interventional studies and PPI use has also been associated with CDI incidence and recurrence in epidemiological studies. We evaluated the ability of a high dose of the PPI omeprazole to alter the murine intestinal microbiota and disrupt colonization resistance to *C. difficile*. We found daily PPI treatment had minimal impact on the murine fecal microbiota and was not enough to promote *C. difficile* colonization after challenge. Further studies are needed to determine whether other factors such as the composition of the starting bacterial community, comorbidities, and use of additional medications contribute to the association between PPIs and CDIs seen in humans.

\newpage

## Introduction
Antibiotics have a large impact on the intestinal microbiome and are a primary risk factor for developing *Clostridium difficile* infections (CDIs) [@Schubert2015]. It is less clear whether other human medications that impact the microbiota also influence *C. difficile* colonization resistance. Multiple epidemiological studies have suggested an association between proton pump inhibitor (PPI) use and incidence or recurrence of CDIs [@tariq2017association; @nehra2018proton; @Naito2018; @Elias2019]. There have also been a number of large cohort studies as well as interventional clinical trials that have demonstrated specific alterations in the intestinal microbiome were associated with PPI use [@Imhann2017; @Naito2018]. PPI-associated microbiota changes have been attributed to the ability of PPIs to increase stomach acid pH which may promote the survival of oral and pathogenic bacteria [@Imhann2017; @Naito2018]. Human fecal microbiota changes with PPI use include increases in Enterococcaceae, Lactobacillaceae, Micrococcaceae, Staphylococcaceae and Streptococcaceae and decreases in Ruminococcaceae [@Imhann2017].    
Unfortunately, most of the studies suggesting a link between PPIs and *C. difficile* were retrospective and did not evaluate the microbiome [@tariq2017association; @nehra2018proton; @Elias2019]. Thus, it is unclear whether the gastrointestinal microbiome changes associated with PPI use play a role in the association between PPIs and CDI incidence or recurrence. Additionally, epidemiological studies have a limited capacity to address potential confounders and comorbidities in patients that were on PPIs and developed CDIs or recurrent CDIs [@Elias2019; @tariq2017association].  Here, we evaluated the impact of a daily high dose PPI treatment on the murine microbiome and susceptibility to *C. difficile* colonization in relation to clindamycin, an antibiotic that perturbs the microbiome enough to allow *C. difficile* to colonize but is mild enough that *C. difficile* is cleared within 10 days [@Jenior2018].

**Murine fecal microbiomes were minimally affected by PPI treatment **
To test if PPI treatment alters the microbiome and promotes susceptibility to CDIs, we gavaged mice daily with a high dose of omeprazole for 7 days before *C. difficile* challenge (Figure 1A). The fecal bacterial communities of PPI-treated mice were compared to a group of mice that received the antibiotic, clindamycin, and a group that received both PPI and clindamycin treatment using 16S rRNA gene sequencing. A principle coordinates analysis (PCoA) of the Bray-Curtis distances over the initial 7 days of treatment revealed the bacterial communities of PPI-treated mice remained relatively unchanged (Figure 1B). Enterococcaceae, Lactobacillaceae, Micrococcaceae, Staphylococcaceae, Streptococcaceae, and Ruminococcaceae are all familes that have previously been impacted by PPIs in human studies, but we observed no significant fluctuations in the stool of our PPI-treated mice throughout the course of the 16-day experiment (Figure 1C, Figure S1). In contrast, the microbiomes from the 2 groups of mice that received clindamycin started to shift away from the PPI-treated mice and the samples from previous days one day after treatment (Figure 1D). These results demonstrated that high dose PPI treatment alone had no significant impact on the murine fecal bacterial community.

**PPI-treatment did not promote susceptibility to *C. difficile infection* in mice**
Next, we examined whether high dose PPI treatment altered susceptibility to *C. difficile* infection in mice. After 7 days of PPI treatment or 1 day after clindamycin treatment, mice were challenged with *C. difficile* 630 spores. While all 4 of the clindamycin-treated mice were colonized with *C. difficile*, all of the PPI-treated mice were resistant to *C. difficile* colonization (Figure 2A). Interestingly, only 1 cage of the PPI and clindamycin-treated mice were colonized, while the other cage was resistant (Figure 2A). PCoAs of the bacterial communities from the 3 groups of mice after *C. difficile* challenge, showed that the greatest shifts in bacterial communities occurred in the clindamycin-treated mice (Figure 2B). Within 5 days, all of the mice had cleared *C. difficile*, suggesting there was no difference in rate of clearance for the mice that were initially colonized with *C. difficile* despite continuing to receive PPI treatment throughout the course of the experiment. Our results suggest that PPI treatment alone had no effects on bacterial community resistance to *C. difficile* colonization in mice. Instead most of the differences between our 3 treatment groups appear to be driven by clindamycin administration and included decreased *Alistipes*, *Barnesiella*, Porhyromonadaceae, Ruminococcaceae, taxa previously found to be altered in clindamycin-treated mice that were challenged with *C. difficile* [@Schubert2015]. These findings demonstrated that high dose PPI treatment did not promote susceptibility to *C. difficile* colonization.

## Discussion and Conclusions
Our findings that PPI treatment had minimal impact on the fecal microbiome were comparable to another PPI mouse study that indicated PPIs had more of an effect on the small intestinal microbiota compared to the fecal microbiota [@Yasutomi2018]. The same group demonstrated PPI treatment increased the stomach pH in mice [@Yasutomi2018], which may improve survival of bacteria passing through the stomach. We did not find significant changes for the taxa observed to be significantly impacted by PPI use in human studies. However, 3 of the families that typically increase were absent or at low abundance in the mice from the beginning and may be one contributing factor to our observation that PPIs had no significant effects on bacterial taxa. Additionally, some of the significance of PPI associations in human interventional trials appears to be driven by a handful of specific taxa with overall differences on a PCoA plot difficult to distinguish [@Freedberg2015, @Imhann2015]. Also, most of the studies to date compared different individuals (PPI users versus non-users) or the same individuals before and after treatment [@Naito2018; @Imhann2017], unfortunately these limited microbiota snapshots may be skewed by day-to-day microbiota variations [@LlornsRico2019] that would be revealed with more frequent longitudinal sampling.   
ALthough there have been a few *C. difficile* mouse model studies that have demonstrated PPIs have some effect on CDIs with or without additional antibiotic treatment[@Kaur2007; @kaur2011effect; @hung2015proton], there were some key methodological differences between these studies and our own. One group administered 0.5 mg/kg of the PPI lansoprazole daily for 2 weeks to mice and then challenged with *C. difficile* demonstrated that PPI treatment alone resulted in detectable *C. difficile* in the stool 1 week after challenge, but also showed there was detectable *C. difficile* in mice not treated with antibiotics [@Kaur2007; @kaur2011effect]. The presence of *C. difficile* in mice that were not treated with either antibiotics or PPIs could be attributed to the higher dose of vegetative cells (10^8^ CFU) used to challenge the mice and may partly explain the observation of *C. difficile* in PPI-treated mice [@Kaur2007; @kaur2011effect]. We have previously shown mice from our colony that were not given antibiotics were resistant to *C. difficile* 630 when challenged with 10^3^ spores [@Jenior2017]. The other group also used a higher dose of 10^6^ spores or 10^7^ vegetative cells to challenge antibiotic-treated mice or mice treated with both the PPI esomeprazole (40mg/kg dose) for 2 days and antibiotics and demonstrated the antibiotic/PPI-treated mice developed more severe CDIs [@hung2015proton].  
Our study extended previous work examining PPIs and *C. difficile* in mice to examine the contribution of the intestinal microbiota. We found the PPI omeprazole had minimal impact on the murine intestinal microbiota and in contrast to previous work, PPIs did not promote *C. difficile* colonization. 16S rRNA sequencing suggested that *Streptococcus* and *Enterococcus* are rare genera in our C57BL/6 mouse colony. These genera could be important contributors to the associations between PPIs and CDIs in humans, and could be a contributing factor to our observation that PPI treatment had no effect on *C. difficile* colonization in our CDI mouse model. There could also be differences at the species level for taxa that are altered with PPIs in humans compared to mice. Other murine starting communities or the addition of human PPI treatment-associated strains may be needed to determine whether PPIs can alter taxa and subsequently promote CDIs.  
Factors such as age, body mass index, comorbidities, and use of other medications in human studies may also be contributing to the association between PPIs and CDI incidence or recurrence. For example, NSAID treatment can increase CDI severity in mice [@Maseda2019] and PPIs are sometimes used in combination with NSAIDs [@nehra2018proton]. This study addressed the impact of PPIs with or without antibiotics on a murine model of CDI, and found PPIs did not promote *C. difficile* colonization. Future studies are needed to determine whether age, other comorbidities and bacterial strains that are less common in mice can increase the risk of CDIs or recurrent CDIs when combined with PPI treatment.   

## Acknowledgements

This research was supported by NIH grant U01AI12455. We would also like to thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the infrastructure and support for performing our mouse expeirments. The authors are also thankful to members of the Schloss lab for helpful discussions throughout the process of designing the experiment, analyzing the results and drafting of the manuscript.  

\newpage

## Materials and Methods

**Animals** All mouse experiments were performed with 7- to 12-week-old C57BL/6 male and female mice. Each experimental group of mice was spit between 2 cages with 2-3 mice housed per cage and male and female mice were housed separately. All animal experiments were approved by the University of Michigan Animal Care and Use Committee (IACUC) under protocol number PRO00006983.

**Drug treatments**
Omeprazole (Sigma Aldrich) was prepared in a vehicle solution of 40% polyethylene glycol 400 (Sigma-Aldrich) in phosphate buffered saline. Omeprazole was prepared from 20 mg/mL frozen aliquots and diluted to an 8mg/mL prior to gavage. All mice received 40 mg/kg omeprazole or vehicle solution once per day through the duration of the experiment with treatment starting 7 days before *C. difficile* challenge (Figure 1A). One day prior to *C. difficile* challenge, 2 groups of mice received an intraperitoneal injection of 10 mg/kg clindamycin or sterile saline vehicle. All drugs were filter sterilized through a 0.22 micron syringe filter before administration to animals.

***C. difficile* infection model**
Mice were challenged with *C. difficile* 630 seven days after the start of omeprazole treatment and one day after clindamycin treatment. Mice were challenged with 10^3^ spores in ultrapure distilled water. Stool samples were collected for 16S rRNA sequencing or *C. difficile* CFU quantification throughout the duration of the experiments at the indicated timepoints (Figure 1A). Samples for 16S rRNA sequencing were flash frozen in liquid nitrogen and stored at -80&deg;C until DNA extraction, while samples for CFU quantification were transferred into an anaerobic chamber and serially diluted in PBS. Diluted samples were plated on TCCFA (taurocholate, cycloserine, cefoxitin, fructose agar) plates and incubated at 37&deg;C for 24 hours under anaerobic conditions to quantify *C. difficile* CFU.

**16S rRNA gene sequencing** DNA for 16S rRNA gene sequencing was extracted from 10-50 mg fecal pellet from each mouse using the DNeasy Powersoil HTP 96 Kit (Qiagen) and an EpMotion 5075 automated pipetting system (Eppendorf). The V4 hypervariable region of the 16S rRNA gene was amplified with Accuprime Pfx DNA polymerase (Thermo Fisher Scientific) using previously described custom barcoded primers [@Kozich2013]. The 16S rRNA amplicon library was sequenced with the MiSeq (Illumina). Amplicons were cleaned up and normalized with the SequalPrep Normalization Plate Kit (ThermoFisher Scientific) and pooled amplicons were quantified with the KAPA library quantification kit (KAPA Biosystems).

**16S rRNA gene sequence analysis**Mothur (v1.40.5) was used for all sequence processing steps [@Schloss2009] using the previously published protocol [@Kozich2013].  R (v.3.5.1) was used to generate figures and perform statistical analysis.

**Statistical Analysis** To assess for differences in relative abundance in families and genera across our 3 different treatment groups (Clindamycin, Clindamycin + PPI, and PPI), we used a Kruskal-Wallis test with a Benjamini-Hochberg correction for multiple comparisons.

**Code availability** The code for all sequence processing and analysis step as well as an Rmarkdown version of this manuscript is available at https://github.com/SchlossLab/Tomkovich_PPI_XXXX_2019.

**Data availability** The 16S rRNA sequencing data have been deposited in the NCBI Sequence Read Archive (Accession no. _________)

\newpage

## References

<div id="refs"></div>

\newpage

## Figures
\includegraphics{figure_1.pdf}
**Figure 1. PPI treatment had minimal impact on the murine fecal microbiota** A. Mouse experiment timeline and logistics. Stools for 16S rRNA sequencing analysis were collected on the days that are labeled (Day -7, -5, -3, -1, 0, 1, 2, 3, 4, 5, 7, 9). B. Principal Coordinates Analaysis (PCoA) of Bray-curtis distances from stool samples for all treatment groups during the intial 7 days of treatment with the PPI omeprazole or vehicle cluster together and do not chage much over time. C. Relative abundances of families previously associated with PPI use in humans do not change much over the 16-day course of treatment with PPIs (7 days before *C. difficile* challenge through 9 days post *C. difficile* challenge). D. PCoA of stool samples collected before and 1 day after antibiotic treatment, demonstrating the fecal microbiota of mice treated with clindamycin starts to separate from the rest of the samples.

\newpage
\includegraphics{figure_2.pdf}
**Figure 2. PPI treatment alone does not promote CDIs in mice** A. C. difficile CFUs/g stool measured each day post C. difficile challenge for clindamycin, clindamycin/PPI, and PPI-treated mice. Lines represent the mean CFU/g for each treatment group while points represent CFU/g for individual mice within each group. B. PCoA of stool samples collected after antibiotic treatments. C. Genera significantly associated with treatment groups for samples across all sequenced timepoints. D. Families significantly associated with treatment groups for samples across all sequenced timepoints. C,D data were analyzed by Kruskal-Wallis test with a Benjamini-Hochberg correction for multiple comparisons.

\newpage
\includegraphics{figure_s1.pdf}
**Figure S1. Previous human PPI-associated families fluctuate over time with no overall trend in either direction** Relative abundance over time for Lactobacillaceae (A) and Ruminococcaceae (B), 2 of the PPI-associated familes from human PPI studies across all 3 treatment groups. Each point represents the relative abundance for an individual mouse 
